capstone-rs/src/constants.rs

```diff
@@ -232,69 +232,69 @@ define_cs_enum_wrapper!(
         => Mode = cs_mode
     ]
     /// 32-bit ARM
-    => Arm = CS_MODE_ARM;
+    => Arm = { cs_mode::CS_MODE_ARM };
```

Comment:
to avoid the need to write `cs_mode::` many times, can we instead modify the `define_cs_enum_wrapper!` macro to include the enum automatically?

Reply:
The problem is that, e.g., `CS_OPT_SYNTAX_INTEL`, which is also used with this macro, is not generated as an enum member:

```
pub mod cs_opt_value {
[...]
    pub const CS_OPT_SYNTAX_INTEL: Type = 1;
```

We could probably indicate to the macro that this is the case, but I thought the extra complexity would not be worth it.